frontend/src/concepts/notebooks/StartNotebookModal.tsx

```diff
@@ -0,0 +1,312 @@
+import * as React from 'react';
+import {
+  Alert,
+  AlertVariant,
+  Content,
+  Flex,
+  FlexItem,
+  List,
+  ListItem,
+  ModalBody,
+  ModalFooter,
+  Panel,
+  PanelMain,
+  ProgressStep,
+  ProgressStepper,
+  ProgressStepVariant,
+  Stack,
+  StackItem,
+  Tab,
+  Tabs,
+  TabTitleText,
+} from '@patternfly/react-core';
+import { Modal, ModalVariant } from '@patternfly/react-core/deprecated';
```

Comment:
Is there a reason we import the deprecated Modal here over the v6 modal?

Reply:
The modal was already built with the deprecated version. I didn't update as there are lots of instances still using the deprecated version. I have updated it now.